src/DataBase/Models/Expenses.cs

```diff
@@ -15,6 +16,26 @@ public class Expenses
         public bool IsMonthly { get; set; }
         public int Importance { get; set; }
 
+        public Expenses() {}
+
+        public Expenses(Entry entry, int userId)
+        {
+            UserId = userId;
+            Amount = entry.Amount;
+            Title = entry.Title;
+            Date = entry.Date;
+            IsMonthly = entry.IsMonthly;
+            Importance = entry.Importance;
+        }
+
+        public void Edit(Entry newEntry)
+        {
+            Amount = newEntry.Amount;
+            Title = newEntry.Title;
+            Date = newEntry.Date;
+            IsMonthly = newEntry.IsMonthly;
+            Importance = newEntry.Importance;
+        }
```

Comment:
Since I'm not very familiar with DB `Edit` is clear to me but that think without name is kinda interesting what is it?
I can see same thing with incomes, but no point commenting same thing.

Reply:
I'm sorry but I'm not sure I understand the question, could you rephrase please?

Reply:
I think I've got it. My question was what is this method `public Expenses(...` And I think it's constructor :) But at that time I was confused so...

Reply:
Yeah it's one of the constructors 👍